src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
The name of the interface 'Area' seems to be using a noun which is good, but since it represents an action (calculating area), it would be more appropriate to name it something like 'AreaCalculator' to reflect that it's providing a behavior rather than being an entity.